pykern/pkcli/ci.py

```diff
@@ -73,6 +64,48 @@ def run():
     from pykern.pkcli import fmt, test
     from pykern import pkio
 
+    check_eof_newline()
     check_prints()
     fmt.diff(pkio.py_path())
     test.default_command()
+
+
+def _check_files(case, check_file):
+    def _error(m):
+        pkcli.command_error("{}: {}", case, m)
+
+    d = PKDict(
```

Comment:
This dictionary could be a global constant like `_PRINT`, and it would simplify the code here.

Reply:
Since this dict is only used in this function, I don't understand why it's better to make it global. It makes this part of the code look cleaner but seems to me like it's actually more complicated by spreading this logic to different parts of the file.

Reply:
Good point. I think the thing I see is a long function which immediately makes it more difficult for me to read. If I see a constant at the top of the file, it's an abstraction (similar to a separate function) that I know is constant so in general, I ignore it unless I need to know the details. This then simplifies the function so the business logic is clearer. Feel free to ask others for opinions.

Reply:
a9e80a0beda32d4163def682c4cffe18637e6076